sdlib/d/gc/extent.d

```diff
@@ -545,6 +545,20 @@ public:
 		return gcCycle == gcWord.load();
 	}
 
+	ulong* getMarksDense(bool clearOutlines = false) {
```

Comment:
bool parameters are usually a bad idea (you have no idea at the call site what they are about.

You can have a `getmarkDense` and a `getMarkDenseAndClearOutline` method that both forward to the same implementation.

Reply:
I switched to a bool parameter, but just for implementation. Is that OK? I also used a template parameter to avoid extra runtime checks.